hosted/gke/p1/p1_provisioning_test.go

```diff
@@ -130,6 +130,52 @@ var _ = Describe("P1Provisioning", func() {
 		Expect(err.Error()).To(ContainSubstring("not found"))
 	})
 
+	It("should be able to create a cluster with CP K8s version v-XX and NP K8s version v-XX-1 should use v-XX for both CP and NP", func() {
+		testCaseID = 33
+
+		k8sVersions, err := helper.ListSingleVariantGKEAvailableVersions(ctx.RancherAdminClient, project, ctx.CloudCred.ID, zone, "")
+		Expect(err).To(BeNil())
+		Expect(len(k8sVersions)).To(BeNumerically(">=", 2))
+		npK8sVersion := k8sVersions[0]
```

Comment:
```suggestion
		npK8sVersion := k8sVersions[1]
		cpK8sVersion := k8sVersions[0]
```
Since CP K8s version v-XX & NP K8s version v-XX-1 ?

Reply:
It was as you suggested initially, but then I figured it would be better to test the other way around to ensure that the CP k8s version is taken into account. I will modify the test title instead, here and on qase.